README.md

```diff
@@ -5,3 +5,10 @@ Some trouble shooting items for continued development
 
 * Updating MyST: `pip install -U mystmd`
 * After updating, `myst clean --all` to clean cached templates
+
+
+## License
+
+All code and text in this repository is free software: you can redistribute it
+and/or modify it under the terms of the MIT License.
+A copy of this license is provided in [LICENSE](LICENSE).
```

Comment:
Should we also mention that content is creative commons (CC-BY-4.0)? There is a lot of text (or maybe we can simply include this in the footer of the website?)

Reply:
Totally. I think we should use CC-BY for text and figures, except where explicitly stated.

Does it make sense to use CC-BY as the license in the Myst configuration then?